sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -335,33 +333,53 @@ private[sbtplugin] object ScalaJSPluginInternal {
       scalaJSConfigSettings
   )
 
-  private val scalaJSTestFrameworkSettings = Seq(
+  val scalaJSTestSettings: Seq[Setting[_]] = (
+      scalaJSConfigSettings
+  ) ++ Seq(
+      /* Always default to false for scalaJSUseMainModuleInitializer in testing
+       * configurations, even if it is true in the Global configuration scope.
+       */
+      scalaJSUseMainModuleInitializer := false,
+
+      // Use test module initializer by default.
+      scalaJSUseTestModuleInitializer := true,
+
+      scalaJSModuleInitializers ++= {
+        if (scalaJSUseTestModuleInitializer.value) {
+          if (scalaJSUseMainModuleInitializer.value) {
+            throw new MessageOnlyException("You may only set one of " +
+                "scalaJSUseMainModuleInitializer and scalaJSUseTestModuleInitializer to true")
```

Comment:
Include `in $configName` in the message where `val configName = configuration.value.name`?